include/boost/graph/isomorphism.hpp

```diff
@@ -81,17 +97,17 @@ namespace detail
         friend struct compare_multiplicity;
         struct compare_multiplicity
         {
-            compare_multiplicity(Invariant1 invariant1, size_type* multiplicity)
+            compare_multiplicity(Invariant1 invariant1, const multiplicity_map& multiplicity)
             : invariant1(invariant1), multiplicity(multiplicity)
             {
             }
             bool operator()(const vertex1_t& x, const vertex1_t& y) const
             {
-                return multiplicity[invariant1(x)]
-                    < multiplicity[invariant1(y)];
+                return multiplicity.at(invariant1(x))
+                    < multiplicity.at(invariant1(y));
             }
```

Comment:
It's a bug if this function is called with invalid `x` or `y`, right? In which case, we should `assert` that they meet the relevant preconditions and ... I was going to say use member `operator[]`, but that's a bit misleading... so I guess it's member `find` and just assume that the iterator returned is not the end.
But basically, let's not use `at` if we're sure that it should be there.